auditlog_security/models/auditlog_log_line.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import exceptions, models, fields
+
+
+class AuditlogLogLine(models.Model):
+    _inherit = 'auditlog.log.line'
+    _order = "create_date desc"
+
+    user_id = fields.Many2one(
+        'res.users',
+        string="User",
+    )
+    method = fields.Char("Method", related='log_id.method')
```

Comment:
I think let this be a stored, indexed field, so that we can also query on it in future

Reply:
"auditlog_log_line_pkey" PRIMARY KEY, btree (id)
    "auditlog_log_line_field_id_index" btree (field_id)
    "auditlog_log_line_log_id_index" btree (log_id)
    "auditlog_log_line_method_index" btree (method)

  @thomaspaulb the infrastructure is there, whenver we need an index we just extend the migration script and bump it up a version:    in this case: 
                     create column if not exists, create index if not exist (with correct name) , prefill the  column with data from log_id.method ( example)